pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>org.hisp.dhis</groupId>
   <artifactId>api-test-utils</artifactId>
-  <version>1.1.1</version>
+  <version>1.1.2</version>
```

Comment:
just FYI: we set the version in deployment action which is triggered when a new release is created from GH. It might break that action, but i'll fix it if it does.

Reply:
ah ok, thanks, I didn't know